webvtt.html

```diff
@@ -1287,29 +1276,9 @@
 
     </ol>
 
-    <p>A <dfn title="partial WebVTT timestamp">WebVTT timestamp representing a time in seconds and
-    fractions of a second</dfn> is a <a>WebVTT timestamp</a> representing hours <var>hours</var>,
-    minutes <var>minutes</var>, seconds <var>seconds</var>, and thousandths of a second
-    <var>seconds-frac</var>, calculated as follows:</p>
-
-    <ol>
-
-     <li><p>Let <var>seconds</var> be the integer part of the time.</p></li>
-
-     <li><p>Let <var>seconds-frac</var> be the fractional component of the time, expressed as the
-     digits of the decimal fraction given to three decimal digits.</p></li>
-
-     <li><p>If <var>seconds</var> is greater than 59, then let <var>minutes</var> be the integer
-     component of <var>seconds</var> divided by sixty, and then let <var>seconds</var> be the
-     remainder of dividing <var>seconds</var> divided by sixty. Otherwise, let <var>minutes</var> be
-     zero.</p></li>
-
-     <li><p>If <var>minutes</var> is greater than 59, then let <var>hours</var> be the integer
-     component of <var>minutes</var> divided by sixty, and then let <var>minutes</var> be the
-     remainder of dividing <var>minutes</var> divided by sixty. Otherwise, let <var>hours</var> be
-     zero.</p></li>
-
-    </ol>
+    <p class="note">A <a>WebVTT timestamp</a> is always interpreted relative to the <a>current
+    playback position</a> of the media data that the WebVTT file is to be synchronized with, which
+    always starts at 0.</p>
```

Comment:
This is a weird statement by the HTML spec.:

"In the rare case of a media resource that does not have an explicit timeline, “

No, the *common* case is that media is all relatively timed, relative to its start.  It’s *rare* that people embed e.g. SMPTE timecodes…

however, not our problem


> On Jul 20, 2015, at 7:05 , Philip Jägenstedt <notifications@github.com> wrote:
> 
> In webvtt.html:
> 
> > -     digits of the decimal fraction given to three decimal digits.</p></li>
> > -
> > -     <li><p>If <var>seconds</var> is greater than 59, then let <var>minutes</var> be the integer
> > -     component of <var>seconds</var> divided by sixty, and then let <var>seconds</var> be the
> > -     remainder of dividing <var>seconds</var> divided by sixty. Otherwise, let <var>minutes</var> be
> > -     zero.</p></li>
> > -
> > -     <li><p>If <var>minutes</var> is greater than 59, then let <var>hours</var> be the integer
> > -     component of <var>minutes</var> divided by sixty, and then let <var>minutes</var> be the
> > -     remainder of dividing <var>minutes</var> divided by sixty. Otherwise, let <var>hours</var> be
> > -     zero.</p></li>
> > -
> > -    </ol>
> > +    <p class="note">A <a>WebVTT timestamp</a> is always interpreted relative to the <a>current
> > +    playback position</a> of the media data that the WebVTT file is to be synchronized with, which
> > +    always starts at 0.</p>
> 
> The "which always starts at 0" bit isn't guaranteed by the HTML spec I think:
> https://html.spec.whatwg.org/#media-timeline
> 
> —
> Reply to this email directly or view it on GitHub.
> 

David Singer
Manager, Software Standards, Apple Inc.